dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -984,6 +1039,13 @@
           "items": {
             "$ref": "#/definitions/HealthcareEntityLink"
           }
+        },
+        "entityComponentInformation": {
+          "description": "(Optional) Entity component information listing fired components of the extracted entity. This object only applies for custom healthcare.",
```

Comment:
Could this be improved by creating another definition that uses `allOf` original properties here and then add custom-specific properties like `entityComponentInformation` to the mix instead of adding it to a single shared definition?

@heaths What do you think?